src/Runner/TestHandler.php

```diff
@@ -113,7 +113,7 @@ private function initiatePhpVersion($version, PhpInterpreter $interpreter)
 
 	private function initiatePhpIni($value, PhpInterpreter $interpreter)
 	{
-		$interpreter->arguments .= ' -d ' . Helpers::escapeArg($value);
+		$interpreter->addArguments(' -d ' . Helpers::escapeArg($value));
```

Comment:
The space at the beginning looks a bit weird, this should be handled internally by interpreter itself.